ui/src/router/components/form/validation/schema.js

```diff
@@ -247,11 +247,22 @@ const mappingSchema = yup.object().shape({
   route: yup.string().required("Treatment needs to be mapped back to a route"),
 });
 
-const standardEnsemblerConfigSchema = yup.object().shape({
-  route_name_path: yup.string().nullable(),
-  experiment_mappings: yup.array(mappingSchema).nullable(),
-  fallback_response_route_id: validRouteSchema,
-});
+const standardEnsemblerConfigSchema = yup
```

Comment:
When this test fails, where would it be captured in the form? Could you add a screenshot?

Reply:
Oops it wouldn't be captured anywhere as of now. In fact, the test wouldn't ever fail because of the automatic appending of the path prefix.